common/persistence/serialization/interfaces.go

```diff
@@ -23,13 +23,281 @@
 package serialization
 
 import (
+	"time"
+
 	"go.uber.org/thriftrw/wire"
 
 	"github.com/uber/cadence/.gen/go/sqlblobs"
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/persistence"
 )
 
+type (
+	// ShardInfo blob in a serialization agnostic format
+	ShardInfo struct {
+		StolenSinceRenew                      *int32
+		UpdatedAt                             *time.Time
+		ReplicationAckLevel                   *int64
+		TransferAckLevel                      *int64
+		TimerAckLevel                         *time.Time
```

Comment:
I think this should be `int64`

Reply:
In persistenceInterface.go this has been converted to time.Time.